### PR TITLE
chore: release notes for 0.9.0

### PR DESCRIPTION
**Pull request by sfoster1**
Release notes for 0.9.0, take a look!

### CODE REVIEW

File: api/release-notes-internal.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Some things are known not to work, and are listed below. Specific compatibility
 
 ## Big New Things
 ### Robot Control
-- move_labware now requires apiLevel to be at least 2.15. You can now move labware off-deck using the python API by supplying OFF_DECK to the new_location arg.
+- Stall detection is enabled for most moves. You might now get stall detection failures if you stall the robot.
```
Comment (TamarZanzouri) [inline api/release-notes-internal.md:15]:
```suggestion
- Stall detection is enabled for most moves. You might now get stall detection failures if you stall the robot.
- move_labware now requires apiLevel to be at least 2.15. You can now move labware off-deck using the python API by supplying OFF_DECK to the new_location arg.
```

**Review by TamarZanzouri — COMMENTED** [review]
quick change! I can commit

File: app-shell/build/release-notes-internal.md [hunk 2]
```diff
@@ -3,15 +3,15 @@ For more details about this release, please see the full [technical changelog][]
...
 This is 0.8.0, an internal release for the app supporting the Opentrons Flex.
 
 This is still pretty early in the process, so some things are known not to work, and are listed below. Some things that may surprise you do work, and are also listed below. There may also be some littler things of note, and those are at the bottom.
 
 ## New Stuff In This Release
-- USB connectivity! If you connect your Flex via USB, it should show up.
-- LPC supports custom labware again
+- USB connectivity improvements: the app will display an icon next to a USB robot's name, USB robots will disappear properly when disconnected, things of this nature.
+- 96-channel detach improvements
 - Visual fixes and improvements
```
Comment (shlokamin) [inline app-shell/build/release-notes-internal.md:15]: could throw in a line about the sick new protocol receipt toasts on the ODD. wont make u do the whole thing again tho lets just include it in the merge the back later

**Review by shlokamin — APPROVED** [review]

**Comment by sfoster1** [comment]
> quick change! I can commit

best part is I can do it from just the web UI

File: app-shell/build/release-notes-internal.md [hunk 3]
```diff
@@ -3,15 +3,15 @@ For more details about this release, please see the full [technical changelog][]
 
 ---
 
-# Internal Release 0.8.0
+# Internal Release 0.9.0
 
 This is 0.8.0, an internal release for the app supporting the Opentrons Flex.
```
Comment (ecormany) [inline app-shell/build/release-notes-internal.md]:
```suggestion
This is 0.9.0, an internal release for the app supporting the Opentrons Flex.
```
😇

File: app-shell/build/release-notes-internal.md (hunk 2 above)
Comment (sfoster1) [reply app-shell/build/release-notes-internal.md:15]: this one is for the desktop app but i added it for the ODD part of the robot one

File: app-shell/build/release-notes-internal.md [hunk 4]
```diff
@@ -3,15 +3,15 @@ For more details about this release, please see the full [technical changelog][]
...
 
-This is 0.8.0, an internal release for the app supporting the Opentrons Flex.
+This is 0.9.0, an internal release for the app supporting the Opentrons Flex.
 
 This is still pretty early in the process, so some things are known not to work, and are listed below. Some things that may surprise you do work, and are also listed below. There may also be some littler things of note, and those are at the bottom.
 
 ## New Stuff In This Release
-- USB connectivity! If you connect your Flex via USB, it should show up.
-- LPC supports custom labware again
+- USB connectivity improvements: the app will display an icon next to a USB robot's name, USB robots will disappear properly when disconnected, things of this nature.
```
Comment (brenthagen) [inline app-shell/build/release-notes-internal.md]: forgot about the robot card icon actually, we haven't addressed that yet - the frontend usb PR was for a green check on the the robot settings networking tab usb connection section

**Pull request merged**